### DISCUSSION TITLE
Homebridge plugin issue

### DISCUSSION THREAD

**Discussion by mhmatttu** [aqualinkd/AqualinkD#182] [General]
i'm getting the below error in homebridge.  config is correct.  i installed the plugin using the homebridge UI.  thanks in advance.

No plugin was found for the platform "AqualinkD" in your config.json. Please make sure the corresponding plugin is installed correctly.

**Comment by sfeakes** [comment]
Can you post your homebridge config.json as this error is has always been an issue with the config file in the past.

**Comment by mhmatttu** [comment]
{
    "bridge": {
        "name": "Homebridge 5851",
        "username": "0E:CD:A5:E4:58:51",
        "port": 51547,
        "pin": "734-61-915",
        "advertiser": "avahi"
    },
    "accessories": [],
    "platforms": [
        {
            "platform": "Hubitat-MakerAPI",
            "name": "Hubitat MakerAPI",
            "app_url": "http://hubitatip/apps/api/178",
            "access_token": "mytoken",
            "local_ip": "homebridgeip",
            "local_port": 20010,
            "mode_switches": false,
            "temperature_unit": "F",
            "hsm": true,
            "debug": false,
            "logFile": {
                "enabled": true,
                "path": "",
                "file": "",
                "compress": true,
                "keep": 5,
                "size": "10m"
            }
        },
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "refreshToken": "mytoken",
            "platform": "Ring"
        },
        {
            "name": "Camera FFmpeg",
            "cameras": [
                {
                    "name": "Driveway",
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-re -i http://myip/mjpg/Driveway/video.mjpg",
                        "audio": false
                    }
                }
            ],
            "platform": "Camera-ffmpeg"
        },
        {
            "name": "AqualinkD",
            "server": "192.168.1.71",
            "port": "81",
            "mqtt": {
                "host": "192.168.1.71",
                "port": "1883",
                "topic": "aqualinkd"
            },
            "platform": "AqualinkD"
        },
        {
            "name": "Russound",
            "controllers": [
                {
                    "name": "Russound",
                    "ip": "myip",
                    "port": 9621,
                    "zones": 5,
                    "sources": 5
                }
            ],
            "platform": "Russound"
        },
        {
            "name": "Sonos",
            "brightness": true,
            "nameScheme": "% Sonos",
            "service": "speaker",
            "speakers": true,
            "platform": "ZP"
        }
    ]
}

**Comment by sfeakes** [comment]
Change to name to lowercase.


`"name": "AqualinkD",`

to

`"name": "aqualinkd",`

if you used the Homebridge ui to configure, then I must have a bug in it.

**Comment by mhmatttu** [comment]
thank you!  i had to make that change for platform as well but that fixed it.  greatly appreciate the help and this project!
